spec/spec_helper.rb

```diff
@@ -0,0 +1,23 @@
+ENV['RACK_ENV'] = 'test'
+require 'minitest/autorun'
+require 'rack/test'
+require 'nokogiri'
+
+Dir.glob('./app/{controllers}/*.rb').each { |file| require file }
+
+include Rack::Test::Methods
+
+DB.drop_table?(:users)
+DB.create_table?(:users) do
+  primary_key :id
+  Boolean :admin
+  String :fname
+  String :lname
+  String :email
+  String :username
+  String :password_hash
+  String :password_salt
+end
+
+DB[:users].insert(admin: true, fname: "f1", lname: "l1", email: "e1", username: "u1")
```

Comment:
Let's move the creation of admin users to the `users_controller_spec` where it's needed. If we'll use it in another test file, then we can add a helper method for it. The test file should show all the setup that is necessary for the test to run.